QueryBuilder/Query.Aggregate.cs

```diff
@@ -58,17 +58,45 @@ public Query SelectAggregate(string type, IEnumerable<string> columns, Aggregate
         }
 
 
+        /**********************************************************************
+         ** AnyValue                                                         **
+         **********************************************************************/
+        public Query SelectAnyValue(string column, string alias = null)
+        {
+            Method = "select";
+            this.AddComponent("select", new AggregateAnyValueColumn
+            {
+                Alias = alias,
+                Column = column,
+                Distinct = AbstractAggregateColumn.AggregateDistinct.aggregateNonDistinct,
+            });
+            return this;
+        }
+
+        public Query SelectAnyValueDistinct(string column, string alias = null)
```

Comment:
I'm not sure I understand what is this - `SELECT DISTINCT(ANY_VALUE([ColName]))` ? Does that query make sense?